editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5922,12 +5973,8 @@ void CanvasItemEditorViewport::_perform_drop_data() {
 	_remove_preview();
 
 	if (!target_node) {
-		// Without root dropping multiple files is not allowed
-		if (selected_files.size() > 1) {
-			accept->set_text(TTR("Cannot instantiate multiple nodes without root."));
-			accept->popup_centered();
-			return;
-		}
+		// Should already be handled by `can_drop_data`.
+		ERR_FAIL_COND_MSG(selected_files.size() > 1, "Can't instantiate multiple scenes without root.");
```

Comment:
Why did you change it to say "scenes"? Sprites from texture are not scenes.

Reply:
Ah, I used `ERR_FAIL_COND()` here initially. The message was written inside `can_drop_data()`, but also became incorrect after multiple passes of refactoring. Then I finally copied the message here, it's not intentional :joy: